files/firmware_status.py

```diff
@@ -104,6 +108,8 @@
 
 jverlist = json.loads(pr.stdout)
 for verd in jverlist:
+    if verd['version'].endswith('b'):
```

Comment:
can you please fix the condition and consider the cfg setting here?
```diff
-    if verd['version'].endswith('b'):
+    if ignore_beta and verd['version'].endswith('b'):
```

Reply:
every time i use `gh` i confuse myself :see_no_evil: